packages/table/src/table.tsx

```diff
@@ -907,33 +1117,37 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
             renderBodyContextMenu,
             selectedRegionTransform,
         } = this.props;
-        const { selectedRegions, viewportRect, verticalGuides, horizontalGuides } = this.state;
+        const { selectedRegions, viewportRect/*, verticalGuides, horizontalGuides*/ } = this.state;
 
-        const style = grid.getRect().sizeStyle();
+        // const style = grid.getRect().sizeStyle();
         const rowIndices = grid.getRowIndicesInRect(viewportRect, fillBodyWithGhostCells);
         const columnIndices = grid.getColumnIndicesInRect(viewportRect, fillBodyWithGhostCells);
-        const noVerticalScroll = fillBodyWithGhostCells &&
-            grid.isGhostIndex(rowIndices.rowIndexEnd, 0) &&
-            viewportRect != null && viewportRect.top === 0 ||
-            this.hasLoadingOption(loadingOptions, TableLoadingOption.ROW_HEADERS);
-        const noHorizontalScroll = fillBodyWithGhostCells &&
-            grid.isGhostIndex(0, columnIndices.columnIndexEnd) &&
-            viewportRect != null && viewportRect.left === 0 ||
-            this.hasLoadingOption(loadingOptions, TableLoadingOption.COLUMN_HEADERS);
+
+        // const noVerticalScroll = fillBodyWithGhostCells &&
```

Comment:
Add `TODO: FROZEN` comment above each of these commented lines/blocks. That can serve as the `grep`-able cue that something needs to be revisited before merging the feature branch back into `master`.